src/parser.rs

```diff
@@ -712,39 +744,6 @@ impl DocParser {
     doc_entries
   }
 
-  pub fn js_doc_for_span(&self, span: Span) -> Option<String> {
-    let comments = self.ast_parser.get_span_comments(span);
-    let js_doc_comment = comments.iter().rev().find(|comment| {
-      comment.kind == CommentKind::Block && comment.text.starts_with('*')
-    })?;
-
-    let mut margin_pat = String::from("");
-    if let Some(margin) = self.ast_parser.source_map.span_to_margin(span) {
-      for _ in 0..margin {
-        margin_pat.push(' ');
-      }
-    }
```

Comment:
I need to investigate this margin_pat a bit more actually. This function has moved to `swc_util` and I removed the `margin_pat` functionality because I didn't see the need for it and the tests still passed. I will do a blame later to investigate more.